trunk/examples/witness-checking/regression/correctness/case_distinction.c-witness.yml

```diff
@@ -0,0 +1,59 @@
+- entry_type: invariant_set
```

Comment:
Yes, the file ending with `.yaml`  contains a witness in version 0.1 (the ghost-witness format as originally planned on VEWIT), while the file ending with `.yml` contains a witness in the current format 2.1. Since we support both formats, we should also have tests for both.

Reply:
- Is it worth the effort to keep supporting both?
- If we do, I'm strongly against distinguishing them by omitting one letter in the file extension. It makes it really hard for anyone to understand what is going.

Reply:
* We already have the support and since our internal datastructures for witnesses are now independant from the format version, supporting another format version requires only a couple of lines in `YamlWitnessParser` and `YamlWitnessPrinter`. Also: the witnesses from Goblint are currently still in the old format 🙂.
* I agree, the file endings are somehow just a "coincicende". It it much better to include the version in the file name (in some way). I just need to check the test suite for witness validation first, since it depends on the file name.

Reply:
Okay, I just renamed the files to indicate ther format version, s.t. the regression tests still consider the witnesses.